dbflow/src/main/java/com/raizlabs/android/dbflow/sql/queriable/StringQuery.java

```diff
@@ -44,7 +64,41 @@ public Cursor query() {
 
     @Override
     public Cursor query(DatabaseWrapper databaseWrapper) {
-        return databaseWrapper.rawQuery(query, null);
+        String[] selectionArgs = new String[params.size()];
+
+        for (ParamsHolder holder : params) {
+            switch (holder.type) {
+                case NULL:
+                    selectionArgs[holder.index] = null;
```

Comment:
#req referencing holder.index directly here will lead to bugs especially if index is out of range of the query.